devpy/cmds/util.py

```diff
@@ -36,18 +37,38 @@ def get_config():
 
 
 def get_site_packages(build_dir):
+    candidate_paths = []
     for root, dirs, files in os.walk(install_dir(build_dir)):
         for subdir in dirs:
-            if subdir == "site-packages":
-                return os.path.abspath(os.path.join(root, subdir))
-
+            if subdir == "site-packages" or subdir == "dist-packages":
+                candidate_paths.append(os.path.abspath(os.path.join(root, subdir)))
+
+    X, Y = sys.version_info.major, sys.version_info.minor
+
+    site_packages = None
+    if any(f"python{X}." in p for p in candidate_paths):
+        # We have a system that uses `python3.X/site-packages` or `python3.X/dist-packages`
+        site_packages = [p for p in candidate_paths if f"python{X}.{Y}" in p]
+        if len(site_packages) == 0:
+            print(f"No site-packages found in `{build_dir}` for Python {X}.{Y}")
+            sys.exit(1)
```

Comment:
It seems wrong for a utility function to call sys.exit. I would prefer it raise and the script runner catch the exception and exit